### PR TITLE
add onboarding panel for long-press

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: webextension/popup.html [hunk 1]
```diff
@@ -60,6 +60,13 @@ <h3 class="onboarding-title">Always open sites in the containers you want.</h3>
     <a href="#" id="onboarding-done-button" class="onboarding-button">Done</a>
```
Comment (jonathanKingston) [inline webextension/popup.html]: This should be changed to:
```
 <a href="#" class="onboarding-button onboarding-almost-done-button">Next</a>
```

I'm also not seeing the panel-5 ever from my testing.

**Comment by groovecoder** [comment]
Updated to properly show the 5th onboarding panel when clicking "Next" on the 4th.

**Comment by jonathanKingston** [comment]
@groovecoder your code looked fine, I changed them to using Enter handlers... which I don't think currently work as the buttons are not focusable which is annoying.

I changed it to use storage which I think will fix #641

**Comment by jonathanKingston** [comment]
And #381

**Comment by groovecoder** [comment]
I checked your changes, but I can't approve my own PR. 😢 

Using my admin privileges to merge without the official approval.

**Pull request merged**